library/src/main/res/values/library_sugarorm_strings.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="define_sugarorm"></string>
```

Comment:
please add `_int` to the define name. or it will always show up. even if the library is not used.